1-js/01-getting-started/2-manuals-and-specifications/article.md

```diff
@@ -0,0 +1,40 @@
+# Manuais e especificações
+
+Este livro é um *tutorial*. Ele tem como objetivo lhe ajudar a aprender gradualmente a linguagem. Mas uma vez que você esteja familiarizado com o básico, irá precisar de outros recursos.
+
+## Especificação
+
+A [Especificacão ECMA-262](https://www.ecma-international.org/publications/standards/Ecma-262.htm) contém as mais profundas, detalhadas e formalizadas informações sobre JavaScript. Ela define a linguagem.
+
+Mas estando num formato mais formal, fica difícil de entender à primeira vista. Então se você precisa da mais confiável fonte de informações sobre os detalhes da linguagem, está no lugar certo. Mas não é para o uso cotidiano.
+
+Uma nova versão dessa especificação é lançada todos os anos. Entre estes lançamentos, a sua última versão de esboço se encontra em <https://tc39.es/ecma262/>.
+
+Para ler sobre as mais novas funcionalidades (*bleeding-edge features*), incluindo as que estão em fase de padronização (também chamadas de "estágio 3"), veja suas propostas em <https://github.com/tc39/proposals>.
+
+E mais, se você está desenvolvendo para browsers, há outras especificações que cobrem esta demanda na [segunda parte]  **(what is the link it should redirect?)** do tutorial.
```

Comment:
Please,

 - '... na [segunda parte](https://javascript.info/browser-environment) do tutorial.'

Here's again the link:
`https://javascript.info/browser-environment`

Reply:
Sorry, I didn't found the link the last time I asked you.
But, this links isn't to the english version?

Reply:
I took the link from the website English version.

Reply:
Will this work to the translated site, @iliakan?